src/components/app.js

```diff
@@ -21,12 +21,19 @@ export default React.createClass({
     this.setState({user});
   },
   render() {
-    //return <Circle user={this.state.user} />;
+    return <Circle user={this.state.user} onPress={ () => {
+        if (!this.currentUser.ref) return <View />;
```

Comment:
This will just leave a blank screen, but the component should handle this internally anyway so leave it and I'll sort it out.